src/v/kafka/server/handlers/api_versions.h

```diff
@@ -14,7 +14,9 @@
 
 namespace kafka {
 
-struct api_versions_handler : public handler<api_versions_api, 0, 2> {
+struct api_versions_handler : public handler<api_versions_api, 0, 3> {
+    static constexpr api_version min_flexible = api_version(3);
```

Comment:
in the future should this be added to the types that the code generator produces?

Reply:
hmm, i was thinking that kafka::handler could contain an alias. but this seems like something we should punt to later. this solution works just fine.